package.json

```diff
@@ -17,7 +17,8 @@
     "check-format": "prettier --check .",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "prepare": "husky install"
+    "prepare": "husky install",
```

Comment:
if you place the new script after check-format , the diff will be reduced to a single file. It also better fits there.